Packs/Darktrace/Integrations/DarktraceMBs/DarktraceMBs.py

```diff
@@ -8,6 +8,7 @@
 from datetime import datetime, timezone
 from typing import Any, cast
 from collections.abc import Mapping
+from json import JSONDecodeError
```

Comment:
```suggestion
```
Use `json.JSONDecodeError` instead of importing `JSONDecodeError` directly to avoid unnecessary imports and keep the source of the exception explicit.

Reply:
this change has been implemented